E2a-TRAS/SP01/README.md

```diff
@@ -0,0 +1,272 @@
+# Test instructions
+
+| Documentation Code   | Test Title                             | Exchange Code | Test Code | Author          | Data Owner | Version | Date       |
+|----------------------|----------------------------------------|---------------|-----------| ----------------|------------|---------|------------|
+| IFC4.3AbRV_E2a_SP01  | 	(RFI) Track super-structure for single track | E2a (TRAS)    | SP01      | Evandro Alfieri | RFI        | 1.0     | DD.MM.YYYY |
+
+
+
+
+## Summary (Intent)
+
+This test case addresses the **export** of the required IFC entities for the exchange of **track super-structure** information.
+
+This test case focuses only on track super-structure (green in picture below). For the track sub-structure part (red in picture below), refer to test case **IFC4.3AbRV_E2a_SB01**, of which SP01 is an extension.
+
+Snippet:
+<img src="./Dataset/TrackCrossSection_snippet.png" width="500"/>
+
+The test also includes the association of some track products (rails, sleepers, ballast, etc.) to: i) the required model breakdown elements (aka, **spatial structure containment**); ii) the required materials (aka, **material association**).
+
+- Refer to [Test Case Imports](#Test-Case-Imports) to know the prerequisites for the present test.
+
+- The [Expected Results](#Expected-Results) section lists the material that will be used to assess the fulfilment of capabilities.
+
+- :zap: This is a test-driven process: refer to the [Validation Criteria](#Validation-Criteria) to understand what is required by the test :zap:
+
+
+
+
+
+## Itemised Roots
+:construction: **under construction, check CT for product shape** :construction:
+
+The Test instruction addresses the import and export of the following IFC Entities & Concept Templates:
+
+<details><summary>IFC Entities</summary>
+
+These entities represent a test-specific subset of the wider AbRV_E2a exchange and the overall AbRV MVD. **The scope of the test shall not be used as a definitive scope of the exchange, or of the MVD**
+
+- Model setup:
+   1. IfcFacilityPart
+- Track domain physical products:
+   1. IfcRail / IfcRailType
+   1. IfcTrackElement
+   1. IfcCourse / IfcCourseType
+</details>
+
+<details><summary>Concept Templates</summary> 
+
+These concept templates represent a test-specific subset of the wider AbRV_Ex exchange and the overall AbRV MVD, that must be correctly exported to meet the validation criteria. **The scope of the test shall not be used as a definitive scope of the exchange, or of the MVD**
+
+- Object Attributes
+   - Object Predefined Type
+- Object definition
+   - Object Typing
+- Product Shape
+   - Product Geometric Representation
+   - Body SectionedSolidHorizontal
+- Object Composition
+   - Spatial Decomposition
+- Object Connectivity
+   - Spatial Containment
+- Object Association
+   - Material Association
```

Comment:
What specific requirements for material are needed here. I could see the possibility to apply:
- Materal Single
- Material Profile Set
- Material Constituent Set

Reply:
added